Automattic/wp-calypso
# ActivityLog: add control to enable text search

Related to p1HpG7-oTm-p2

## Proposed Changes

* Add control to enable posts search by ID, title and author in the Activity Log.

![CleanShot 2023-09-28 at 16 12 21](https://github.com/Automattic/wp-calypso/assets/1488641/273a309f-d25d-4424-8fb8-335c055fa597)

## Testing Instructions
* Spin up a Calypso live branch
* Navigate to the Activity Log
* Ensure the search box is visible and try searching your posts and play around with the filters. You must enter data and press ENTER to start filtering.
* Ensure this search box is visible on Calypso Blue and Jetpack Cloud.

## Pre-merge Checklist
